app/src/main/java/com/artemzin/qualitymatters/models/PicassoBitmapClient.java

```diff
@@ -0,0 +1,21 @@
+package com.artemzin.qualitymatters.models;
+
+import android.widget.ImageView;
+import com.squareup.picasso.Picasso;
+
+import javax.inject.Inject;
+
+public class PicassoBitmapClient implements NetworkBitmapClient {
+
+  private final Picasso picasso;
+
+  @Inject
```

Comment:
I'd just like to point out that Dagger's [official testing documentation](http://google.github.io/dagger/testing.html) says to **NOT** use Dagger for unit testing. You should call the @Inject annotated constructor manually and pass in fake or mocked dependencies directly. This also has the added benefit of removing all of the "magic" of the object instantiation.